articles/email/index.md

```diff
@@ -4,50 +4,49 @@ url: /email
 
-### From Address
+### *From Address*
 
-The **From address** is the address the end user will see when receiving an email from Auth0.
+The **From address** is the address users will see as the sender when receiving an email from Auth0.
 
 If you haven't configured a **From address** for your emails, the following will occur:
 
 * If your email templates are not customized, they will be sent from `no-reply@auth0.com`.
 * Otherwise, they will be sent from the email address of the first owner of your Auth0 account.
 
-> Because of security reasons, it is not possible to send customized emails from any `@auth0.com` address.
-> Appliance users can configure a similar domain blacklist as well.
+**NOTE:** Because of security reasons, it is not possible to send customized emails from any `@auth0.com` address. Appliance users can configure a similar domain blacklist.
 
 This field supports the following macros:
 
 - `{application.name}`
 - `{connection.name}`
 
-With this macros you could set the display name of the **From address** to something specific for the current application the user signed up for. Eg: `{application.name} <support@fabrikamcorp.com>`
+With these macros, you could set the display name of the **From address** as something specific to the current application for which the user signed up. Eg: `{application.name} <support@fabrikamcorp.com>`
 
-In order to avoid your emails ending up in the user's junk folder we recommend adding the required [SPF](http://en.wikipedia.org/wiki/Sender_Policy_Framework) and [DKIM](http://en.wikipedia.org/wiki/DKIM) DNS records, allowing Auth0 to send the emails on your behalf and digitally sign the messages.
+To avoid emails ending up in your users' junk folder, add the required [Sender Policy Framework (SPF)](http://en.wikipedia.org/wiki/Sender_Policy_Framework) and [DomainKeys Identified Mail (DKIM)](http://en.wikipedia.org/wiki/DKIM) DNS records, allowing Auth0 to send digitally-signed emails on your behalf.
 
-Additionally, when SPF and DKIM are not configured you users might see the **From address** with "on behalf of" depending on the email client they're using:
+When SPF and DKIM are not configured, users may see the **From address** as "on behalf of", depending on their email client:
 
-> MyApp support@mail128-21.atl41.mandrillapp.com on behalf of MyApp support@fabrikamcorp.com
+`MyApp support@mail128-21.atl41.mandrillapp.com on behalf of MyApp support@fabrikamcorp.com`
 
 #### SPF Configuration
 
-The SPF is configured by adding a TXT record to your domain. The hostname should be set to @ (or empty, depending on the provider). If this is the first time you're setting up an SPF record you'll need to declare it as follows:
+The SPF is configured by adding a TXT record to your domain. The host name should be set to @ (or empty, depending on the provider). If this is the first time you are setting up an SPF record, you will need to declare it as follows:
 
 ```
 v=spf1 include:spf.auth0.com ~all
```

Comment:
I'm not actually sure. @sebastianrodriguez, do you know if tilde is correct? https://wordtothewise.com/2014/06/authenticating-spf/